misago/users/tests/test_anonymizeoldips.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+import pytz
+
+from django.contrib.auth import get_user_model
+from django.core.management import call_command
+from django.test import TestCase
+from django.utils.six import StringIO
```

Comment:
Imports should be ordered alphabetically. `django.utils.six` should come after `django.utils`.